pkg/agent/cniserver/ipam/ipam_delegator_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipam
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	argtypes "antrea.io/antrea/pkg/agent/cniserver/types"
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"github.com/stretchr/testify/assert"
+)
+
+type networkConf struct {
+	CNIVersion string `json:"cniVersion"`
+}
+
+var (
+	testNetworkConfig, _ = json.Marshal(networkConf{CNIVersion: "0.4.0"})
```

Comment:
Use a variable to replace the hard-coded `CNIVersion`?

Reply:
Done.

Reply:
I mean a variable for "0.4.0", since I saw you use this version many times. Use a variable is easier for future upgrade.

Reply:
Sorry I didn't notice that there's "0.4.0" string in other files, so for the last commit I just changed the `antrea_ipam_test.go`. Now both `ipam_delegator_test.go` and `ipam_service_test.go` were changed to `testCNIVersion` too